### ISSUE TITLE
Allow passing concurrent options to ESLint via Angular CLI linting

### ISSUE THREAD

**Issue by csvn** [angular-eslint/angular-eslint#2634]
**Description and reproduction of the issue**

Allow passing the [new concurrent option in ESLint](https://github.com/eslint/eslint/pull/19794) when calling `npx ng lint`. From the current set of options, I can't see any way to pass this option, which I'm interested in trying to speed up linting.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/builder`          | `20.1.1` |
| `ESLint`                           | `9.34.0` |
| `node`                             | `24.5.0` |

```sh
# Please run `npx ng version` in your project and paste the full output here:
Angular CLI: 20.2.0
Node: 24.5.0
Package Manager: npm 11.5.1
OS: win32 x64


Angular: 20.2.1
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, router

Package                              Version
--------------------------------------------
@angular-devkit/architect            0.2002.0
@angular-devkit/core                 20.2.0 (cli-only)
@angular-devkit/schematics           20.2.0
@angular/build                       20.2.0
@angular/cdk                         20.1.5
@angular/cli                         20.2.0
@angular/material                    20.1.5
@angular/material-date-fns-adapter   20.1.5
@schematics/angular                  20.2.0
rxjs                                 7.8.2
typescript                           5.9.2
zone.js                              0.15.1

```

- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by csvn** [comment]
Seems this issue is addressed by #2636, which was released in [v20.3.0](https://github.com/angular-eslint/angular-eslint/releases/tag/v20.3.0) 🎉
